DEVELOPMENT.md

```diff
@@ -109,7 +109,15 @@ endings match the project style.
 
 ## Merging PRs to the codebase
 
-The docs for this repo are hosted by readthedocs.io and can be found [here](https://cleanvision.readthedocs.io/en/latest/). 
+The docs for this repo are hosted by readthedocs.io and can be found [here](https://cleanvision.readthedocs.io/en/latest/).
 Currently, merged PRs *will not* update the docs -- you **need** to do this manually after merging a PR.
```

Comment:
Is this still true? I think PRs merged to main will update docs!

Reply:
No